pkg/consts/constants.go

```diff
@@ -42,6 +42,9 @@ const (
 	LinkTypeIB  = "IB"
 	LinkTypeETH = "ETH"
 
+	DefaultNodeGUID = "0000:0000:0000:0000"
```

Comment:
nit: can you rename to: UninitializedNodeGUID ?

since you cannot set it back to "default "